frontend.rst

```diff
@@ -78,6 +78,26 @@ pre-processing CSS & JS and compiling and minifying assets.
 
 :doc:`Read the Encore Documentation </frontend/encore/index>`
 
+Switch from Asset Mapper
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, new Symfony _webapp_ projects (created with ``symfony new --webapp myapp``)
```

Comment:
In RST, emphasis is with `*` (italics) and `**` (bold) instead of `_` and `*` in Markdown.

So, this should be `*webapp*` ... although maybe we could remove that and just show "... new Symfony webapp projects..." or does that read too bad for native speakers?

Reply:
I removed the emphasis as suggested.